core/build.sbt

```diff
@@ -46,8 +46,6 @@ pomExtra := {
   </developers>
 }
 
-osgiSettings
```

Comment:
I'm not familiar with OSGi, but removing this changes the way the artifacts are published. Instead of a `bundles` folder we get a `jar` folder inside the archive and the OSGi headers seem to be absent from the JAR's manifest.